qa-tests-backend/src/main/groovy/services/ExternalBackupService.groovy

```diff
@@ -51,4 +51,30 @@ class ExternalBackupService extends BaseService {
                 .setS3(s3Config)
                 .build()
     }
+
+    static ExternalBackupOuterClass.ExternalBackup getGCSIntegrationConfig(
```

Comment:
nit: have you thought about static import so we do not need to repeat `ExternalBackupOuterClass.` everywhere?
```
import static io.stackrox.proto.storage.ExternalBackupOuterClass.*
import static io.stackrox.proto.storage.ScheduleOuterClass.*
```

Reply:
How about adding `@CompileStatic` for this class?